core/js/typespecs/functions.ts

```diff
@@ -88,6 +97,21 @@ const invokeFunctionNonIdArgsSchema = z.object({
     .describe(
       "Whether to stream the response. If true, results will be returned in the Braintrust SSE format.",
     ),
+  update_score: z
```

Comment:
this seems a little strange to include in the public API for invoke. Should we make it a hidden argument or a separate endpoint or something?

Reply:
Agreed that this makes sense to maintain internally for now. Will move.